Formula/bgfx.rb

```diff
@@ -0,0 +1,47 @@
+class Bgfx < Formula
+  desc "Cross-platform, graphics API agnostic, rendering library"
+  homepage "https://github.com/bkaradzic/bgfx"
+  url "https://github.com/bkaradzic/bgfx/archive/403b69db7876bc7269f546273bba57c40d2b137e.tar.gz"
```

Comment:
We only build from release tags. Please ask upstream to tag a release so we can package this.

Reply:
@carlocab - Thank you for the detailed review. I am discussing with the bgfx owner if he can tag a release. Hopefully we can get this done soon.